docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,247 @@
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/FrameworkAgreement",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "traceability:1.0"
+}
+```
+
+### Usage Purposes
+
+Purposes are usually part of a Use Case Framework and restrict the purpose the Consumer is privileged to use the
+obtained data for. Unlike a Use Case Framework Constraint, the purposes are NOT checked against VCs, thus necessary
+for a successful negotiation mechanism is
+only [the Consumer's consent to the Offer](working-with-policies.md#consumer-side-odrloffer-in-a-contractrequestmessage).
+Versions for UsagePurpose `rightOperand`s are typically 1-digit. The complete list of usage purposes in Catena-X is
+publicly available in the [CX ODRL Profile](https://github.com/catenax-eV/cx-odrl-profile). The corresponding documents
+are linked on
+the [Catena-X e.V. homepage](https://catena-x.net/en/catena-x-introduce-implement/governance-framework-for-data-space-operations).
+Here's an example from the Use Case Framework Traceability:
+
+| Predefined Policy                | Typically used where? | Predefined Purpose                                                                                                                                        |
+|----------------------------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `cx.core.qualityNotifications:1` | Notification API      | The data can be used for quality analysis to identify and select affected components and to send quality notifications to affected customers or suppliers |
+
+A `odrl:Constraint` referencing this purpose looks like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/UsagePurpose",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "cx.core.qualityNotifications:1"
+}
+```
+
+### Contract References
+
+Contract References by default aren't checked against credentials either. They are a vehicle to refer to contracts that
+are not governed by the Catena-X association - for instance bilaterally. Referencing such a contract's identifier can
+be achieved via an `odrl:Constraint` like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/ContractReference",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "contract-123456789"
+}
+```
+
+### Chaining Constraints
+
+If a Policy is supposed to hold multiple constraints, Data Providers may chain them via a logical AND. This can be
+achieved via an `odrl:and` object encapsulating multiple other `odrl:Constraint`s or entering a list of them
```

Comment:
Formally correct you could say something like:

"...`Rule` classes `constraint` property may hold a `Constraint` or a `LogicalConstraint` that itself holds - in case of a logical AND - an `odrl:and` property with a list of `Cosntraint`s."

but this is really a minor comment...

Reply:
I'd prefer to leave it this way - especially since it also opens the option to 
```json
{
  "action": "use",
  "constraint": [
    {
      "leftOperand": "cx-policy:FrameworkAgreement",
      "operator": "eq",
      "rightOperand": "traceability:1.0"
    },
    {
      "leftOperand": "cx-policy:ContractReference",
      "operator": "eq",
      "rightOperand": "x12345"
    },
    {
      "leftOperand": "cx-policy:UsagePurpose",
      "operator": "eq",
      "rightOperand": "cx.core.industrycore:1"
    }
  ]
}
```

Reply:
which is basically also ODRL and there is no way to not allow this - to my understanding.... and btw: your example would have been my preferred solution / documentation examples.
Even though in my text above, I didn't mention this. I just described the LogicalConstraint that we are using - as kind of mandatory - which it isn't.
but as I said, this comment was a minor one. just ignore it :-)